core/arch/x86/opcode.h

```diff
@@ -1319,6 +1319,13 @@ enum {
     /* 1156 */ OP_ktestq,   /**< IA-32/AMD64 AVX-512 ktestd opcode. */
     /* 1157 */ OP_ktestd,   /**< IA-32/AMD64 AVX-512 ktestd opcode. */
 
```

Comment:
Remove the blank line: see the groupings above which have a label and then contiguous lines for each set.

Reply:
I added AVX-512 VEX and AVX-512 EVEX and kept the blank line if that's ok.

Reply:
You mean you added a label after the blank line?

Reply:
Yes.

Reply:
I assume there are more of these "AVX-512 EVEX"?  Not worth a separate labeled group for just this handful.

Also, if the plan is to alphabetize, will new ones be inserted above these ones?  We can do that for between-releases stuff but it may be worth a comment here telling the reader to not rely on these opcode enum numbers right now b/c they're in flux.

Reply:
> Assume there are more of these "AVX-512 EVEX"

According to my current count, 211 more, yes (xref #1312).

Good point, I'll add a comment.